sparkprs/jira_api.py

```diff
@@ -12,6 +12,37 @@ def get_jira_client():
                                         app.config['JIRA_PASSWORD']))
 
 
+def start_issue_progress(issue):
+    """
+    Given an issue key, e.g. SPARK-6481, mark the issue "In Progress".
+
+    This will only happen if the issue's initial state is "Open".
+    """
+    jira_client = get_jira_client()
+    issue_info = jira_client.issue(issue)
+    status = issue_info.fields.status.name
+    assignee = issue_info.fields.assignee.name if issue_info.fields.assignee else None
+
+    if status != "Open":
+        logging.warn("Could not start progress on JIRA issue {j}. "
```

Comment:
This log message will be a bit chatty in the common case where we call `start_issue_progress` on an issue that's already in progress.  Could you update this so that we return without any logging if the issue is already in-progress and log a message only if the status is in a state other than "In Progress"?